core/object.cpp

```diff
@@ -2123,21 +2123,38 @@ void ObjectDB::cleanup() {
 	if (slot_count > 0) {
 		spin_lock.lock();
 
-		WARN_PRINT("ObjectDB Instances still exist!");
 		if (OS::get_singleton()->is_stdout_verbose()) {
+			WARN_PRINT("ObjectDB Instances still exist!");
+
+			// Ensure calling the native classes because if a leaked instance
+			// has a script overriding nay of those methods, it'd not be OK to
```

Comment:
Typo: "nay".
You can also add a dot at the end of the sentence.

Reply:
Both fixed.